setup/setup.iss

```diff
@@ -69,12 +67,11 @@ Name: "{code:GetGameDir}\logs"; Permissions: users-modify
 
 [Icons]
 Name: "{group}\Dash Faction"; Filename: "{app}\DashFactionLauncher.exe"
-Name: "{commondesktop}\Dash Faction"; Filename: "{app}\DashFactionLauncher.exe"; Tasks: desktopicon
+Name: "{autodesktop}\Dash Faction"; Filename: "{app}\DashFactionLauncher.exe"; Tasks: desktopicon

```

Comment:
have you tried running the installer in non-admin mode? Does it work? It certainly cannot work if there are patches to be applied but I guess we can keep using auto constants...

Reply:
It's not something that I have tried. I just replaced the common constants with the auto ones because it was recommended in the Inno docs: https://jrsoftware.org/ishelp/index.php?topic=consts

I'd imagine that running in non-admin would be very broken due to Program Files folder permissions and other issues...